ui/src/app/applications/components/application-node-info/application-node-info.scss

```diff
@@ -39,4 +39,8 @@
             color: $argo-color-gray-8;
         }
     }
+    &__err_msg {
+        padding-right: 2em;
+        color: #E96D76;
```

Comment:
Use `$argo-failed-color` instead of `#E96D76` for color reference as it already imports colors from `'node_modules/argo-ui/src/styles/config'`.

Reply:
Agree